bucket/iobit-unlocker.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.3.0",
+    "description": "Solution for \"undelete files or folders\" Problems",
+    "homepage": "https://www.iobit.com/iobit-unlocker.php",
+    "license": {
+        "identifier": "Proprietary",
```

Comment:
Wrong identifier. Should be Freeware. See
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#required-properties